lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -312,14 +315,14 @@ export abstract class ClientApplication {
      * object. If no authority set in application object, then default to common authority.
      * @param authorityString - authority from user configuration
      */
-    private async createAuthority(authorityString: string): Promise<Authority> {
+    private async createAuthority(authorityString: string, prefferedAzureRegionOptions?: PrefferedAzureRegionOptions): Promise<Authority> {
         this.logger.verbose("createAuthority called");
         const authorityOptions: AuthorityOptions = {
             protocolMode: this.config.auth.protocolMode!,
             knownAuthorities: this.config.auth.knownAuthorities!,
             cloudDiscoveryMetadata: this.config.auth.cloudDiscoveryMetadata!,
             authorityMetadata: this.config.auth.authorityMetadata!
         };
-        return await AuthorityFactory.createDiscoveredInstance(authorityString, this.config.system!.networkClient!, this.storage, authorityOptions);
+        return await AuthorityFactory.createDiscoveredInstance(authorityString, this.config.system!.networkClient!, this.storage, authorityOptions, prefferedAzureRegionOptions);
```

Comment:
Is this feature applicable to browser? If so please make the changes there as well.